app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -85,46 +69,56 @@ To be eligible for a scholarship, you’ll need at least one of the following:
 * a master’s degree
 * a PhD
 
-In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
+If you achieved a 2:2 in your degree you may get a scholarship if you have enough relevant experience.
+
+Scholarship providers set their own:
 
-#### Benefits
+* eligibility requirements 
+* application deadlines
 
-Scholarships are provided by charitable organisations with a passion for their subject area.
+#### How they're paid
```

Comment:
```suggestion
#### How scholarships are paid
```
Currently there are two identical h4s on the page which is not good for screen reader users